_pages/setup-wallets/paper-advanced.md

```diff
@@ -60,12 +60,7 @@ Expert users only: <https://github.com/mflaxman/human-rng-golang>
 #### Improve Airgap
 
 ##### Option A: Write Data to CD/DVD instead of USB Drive
-DVDs are less able to execute malware on your computer.
-
-##### Option B: Use a QR-Code Airgap
-This is not currently documented.
-Pull requests with clarification are welcome.
-Specter-Desktop can receive this info via QR-code airgap.
+DVDs are less able to execute malware on your computer
```

Comment:
Is this intentional? Having an option A without an Option B is confusing.

I think we still want to list both options, until QR-code adoption is universal (many HW wallets with cameras for example). Some users are on desktops without webcams! Maybe we can make QR-code the preferred option?